tests/www/api/experimental/test_dag_runs_endpoint.py

```diff
@@ -41,6 +41,7 @@ def setUpClass(cls):
     def setUp(self):
         super().setUp()
         app, _ = application.create_app(testing=True)
+        app.config['STORE_SERIALIZED_DAGS'] = False
```

Comment:
Can you use conf_vars context manager/decorator here to avoid side-effect?

Reply:
To generate identical tests, but in different configurations, it is worth using parameterized.

Reply:
Could you give me an example of using "conf vars"? Do you mean AIRFLOW__CONFIG__STORE_SERIALIZED_DAGS global env variable.
Sorry that I did not ask that earlier.

Reply:
Ok, I have found it in another test: @conf_vars({('core', 'enable_xcom_pickling'): 'True'})